File: testresources/java/accuracy/src/domain/direct/violating/DeclarationTypeCastToInnerClass.java
package domain.direct.violating;

import technology.direct.dao.CallInstanceOuterClassDAO;

public class DeclarationTypeCastToInnerClass {
	
	public String getProfileInformation(){
		Object o = (CallInstanceOuterClassDAO.CallInstanceInnerClassDAO) new Object();
		return o.toString();
	}
}

File: src/test/resources/csharp/benchmark/Domain/Direct/Violating/DeclarationTypeCastToInnerClass.cs
namespace Domain.Direct.Violating

{

	using Technology.Direct.Dao;

	public class DeclarationTypeCastToInnerClass
	{
		public void GetProfileInformation()
		{
            object o = (CallInstanceOuterClassDAO.CallInstanceInnerClassDAO)new object();
		}
	}
}
